chapters/functions.tex

```diff
@@ -295,8 +296,8 @@ \section{Pure Modelica Functions}\label{pure-modelica-functions}
 A function having a formal function parameter that is \lstinline!impure! must be marked \lstinline!pure! or \lstinline!impure!.
 
 \begin{nonnormative}
-Comment: The semantics are undefined if the function call of an
-impure function is part of an algebraic loop.
+Comment: The semantics are undefined if the function call of an impure function is part of an algebraic loop.
```

Comment:
This is the only _Comment:_ at the start of a non-normative section.  Since all non-normative content is some sort of content, we should remove it:
```suggestion
The semantics are undefined if the function call of an impure function is part of an algebraic loop.
```